Modules/HR/Entities/ApplicationEvaluationSegment.php

```diff
@@ -15,6 +18,10 @@ public function applicationRound()
     {
         return $this->belongsTo(ApplicationRound::class);
     }
+    public static function newFactory()
```

Comment:
Leave a line space before and after a function
```suggestion

    public static function newFactory()
```